manager/src/main/res/values-ru/string.xml

```diff
@@ -0,0 +1,121 @@
+
+    <!-- Adb -->
+    <string name="dialog_adb_discovery">Поиск службы беспроводной отладки</string>
+    <string name="dialog_adb_discovery_message">Please enable \"Беспроводная отладка\" в \"Настройки для разработчиков\". \"Беспроводная отладка\" автоматически отключается при изменении сети.\n\nПримечание, не отключайте \"Настройки для разработчиков\" или \"Отладка по USB\", иначе Shizuku будет остановлен.</string>
+    <string name="dialog_adb_port">Порт</string>
+    <string name="dialog_adb_invalid_port">Порт - это число от 1 до 65535.</string>
+    <string name="adb_pairing">Сопряжение</string>
+    <string name="dialog_adb_pairing_title">Сопряжение с устройством</string>
+    <string name="dialog_adb_pairing_discovery">Поиск службы сопряжения</string>
+    <string name="dialog_adb_pairing_paring_code">Код сопряжения</string>
+    <string name="paring_code_is_wrong">Неправильный код сопряжения.</string>
+    <string name="cannot_connect_port">Не удается подключить службу беспроводной отладки.</string>
+    <string name="dialog_wireless_adb_not_enabled">Беспроводная отладка не включена.\nОбратите внимание, что до Android 11, чтобы включить беспроводную отладку, необходимо подключение к компьютеру.</string>
+    <string name="dialog_adb_pairing_message">Пожалуйста, для подключения выполните следующие шаги: \"Настройки для разработчиков\" - \"Беспроводная отладка\" - \"Подключение устройства используя код\".\n\nПосле запуска процесса сопряжения вы сможете ввести код сопряжения.</string>
+    <string name="adb_pairing_requires_multi_window">Пожалуйста, сначала войдите в режим разделения экрана (многооконный).</string>
+    <string name="adb_pairing_requires_multi_window_reason">Система требует, чтобы диалог сопряжения всегда был виден, использование режима разделения экрана - это единственный способ сделать это приложение и системный диалог видимыми одновременно.</string>
+    <string name="adb_error_key_store">Невозможно сгенерировать ключ для беспроводной отладки.\nЭто может быть связано с тем, что механизм хранилища ключей этого устройства сломан.</string>
+    <string name="adb_pair_required">Пожалуйста, сначала выполните шаг сопряжения.</string>
+    <string name="development_settings">Настройки для разработчиков</string>
+
+    <!-- Home - Root -->
+    <string name="home_root_title"><![CDATA[Запуск (требуются права суперпользователя)]]></string>
+    <string name="home_root_description"><![CDATA[Кроме того, Shizuku может быть запущена автоматически при загрузке. Если нет, пожалуйста проверьте ограничивают ли система или сторонние инструменты Shizuku.<br>Вы можете обратиться к %s.]]></string>
+    <string name="home_root_button_start">Запуск</string>
+    <string name="home_root_button_restart">Перезапуск</string>
+
+    <!-- Home - Application management -->
+    <string name="home_app_management_title">Управление приложениями</string>
+    <plurals name="home_app_management_authorized_apps_count">

```

Comment:
For locale "ru" (Russian) the following quantities should also be defined: few, many

Reply:
Ok